p2p/src/authenticated/lookup/mod.rs

```diff
@@ -416,23 +416,23 @@ mod tests {
         assert_eq!(state, state2);
     }
 
-    #[test_traced]
+    #[test_traced("INFO")]
```

Comment:
For some reason, stack was exhaused on debug 🤔 (if you print out on debug the spans are huge...maybe related)?

Reply:
Seems to be an issue with what is on `main`, not an issue with this pr.